homeassistant/components/sensor/recorder.py

```diff
@@ -678,7 +676,7 @@ def validate_statistics(
 
     for state in sensor_states:
         entity_id = state.entity_id
-        state_class = state.attributes.get(ATTR_STATE_CLASS)
+        state_class = try_parse_enum(SensorStateClass, state.attributes.get(ATTR_STATE_CLASS))
```

Comment:
My previous suggestion failed `black` 👎 
```suggestion
        state_class = try_parse_enum(
            SensorStateClass, state.attributes.get(ATTR_STATE_CLASS)
        )
```